dump-functions-memory-usage.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Should we stick this into `src/utils`?

Reply:
Didn't know about it. Maybe we could hide the script inside `Makefile` and make it operate on Target names instead.

Reply:
That's definitely another option.